layouts/partials/navbar.html

```diff
@@ -8,52 +8,51 @@
 {{ $isRelease := in site.BaseURL "www.o3de.org" }}
 
 {{ if $isDocs }}
-<!-- TODO: Additional sticky-top is temporary. Working on a better way to stop left nav from scrolling. -->
-<div class="sticky-top">
-<div class="main-navbar navbar navbar-dark navbar-docs sticky-top" role="navigation" aria-label="main navigation">
-  <div class="container-fluid d-flex flex-row justify-content-between">
-      <div class="navbar-brand">
-        <a class="navbar-item" href="{{ $docs }}">
-          {{ if $logo }}
-          {{ $url := printf "img/logos/%s" $logo | relURL }}
-            <img src="{{ $url }}" class="navbar-logo">
-          {{ else }}
-            {{ $title }}
-          {{ end }}
-          <span class="subTitle">LEARN</span>
-        </a>
-      </div>
-
-      <div class="d-flex flex-row align-items-center navbar-docs">
-        <div id="main-nav" class="navbar-primary">
-          {{ partial "docs-menu-loop.html" . }}
+<div class="sticky-top" id="docs-banners">
+  <div class="main-navbar navbar navbar-dark navbar-docs" role="navigation" aria-label="main navigation">
```

Comment:
These two lines contain the only important changes to navbar.html in this PR. The rest is fixing indentation.